tests/installation/user_settings.pm

```diff
@@ -39,7 +39,7 @@ sub run {
     my $max_tries = 4;
     my $retry     = 0;
     do {
-        $self->enter_userinfo(max_interval => ($retry) ? utils::VERY_SLOW_TYPING_SPEED : undef);
+        $self->enter_userinfo(username => get_var('USER_FULLNAME', undef), max_interval => ($retry) ? utils::VERY_SLOW_TYPING_SPEED : undef);
```

Comment:
this means tthat per product/jobgroup, we can end up with at least 1 needle right?... This is going to be a maintenance burden...

Reply:
If they all choose the same name 'bernhard' it should only be one needle

Reply:
Should then be `IGNORE_BSC1122804=1`

```perl
if (get_var(IGNORE_BSC1122804)) {
  $self->enter_userinfo(username => 'bernhard');
  assert_screen('inst-userinfostyped-just-bernhard');
} else {
    my $max_tries = 4;
    my $retry     = 0;
    do {
        $self->enter_userinfo(max_interval => ($retry) ? utils::VERY_SLOW_TYPING_SPEED : undef);
        assert_screen([qw(inst-userinfostyped-ignore-full-name inst-userinfostyped-expected-typefaces)]);
        record_soft_failure('boo#1122804 - Typing issue with fullname') unless match_has_tag('inst-userinfostyped-expected-typefaces');
        $retry++;
    } while (($retry < $max_tries) && !match_has_tag('inst-userinfostyped-expected-typefaces'));
    assert_screen('inst-userinfostyped-expected-typefaces');    # fail if mistyped
}
```

Reply:
+1

Reply:
can you please rebase your PR on latest master , since #9807 already merged and it will be more clear what you trying to achieve